src/app.js

```diff
@@ -1,21 +1,32 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
-
-module.exports = sum;
+const http = require('http');
+const fs = require('fs');
+
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const { pathname } = new URL(req.url, `http://${req.headers.host}`);
+  const pathParts = pathname.slice(1).split('/');
+
+  if (pathParts[0] !== 'file') {
+    res.end('The file path must start with "/file/"');
+
+    return;
+  }
+
+  const pathToFile = pathParts.slice(1).join('/') || 'index.html';
+
+  fs.readFile(`./public/${pathToFile}`, (error, data) => {
+    if (error) {
+      res.statusCode = 404;
+      res.end('404 Error: File not found');
+    }
+
+    res.end(data);
```

Comment:
You are ending the response before setting the Content-Type header. This could lead to unexpected behavior on the client-side, as the client may not know how to interpret the data. You should set the Content-Type header before calling res.end(data).